CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/state/AnalyticViewState.java

```diff
@@ -65,7 +66,11 @@ public void setCurrentAnalyticQuestionIndex(final int currentAnalyticQuestionInd
 
     public void addAnalyticQuestion(final AnalyticQuestionDescription<?> question, final List<SelectableAnalyticPlugin> selectablePlugins) {
         activeAnalyticQuestions.add(question);
-        activeSelectablePlugins.add(selectablePlugins);
+        for (SelectableAnalyticPlugin plugin : selectablePlugins) {
```

Comment:
What's the advantage of this, isn't it functionally equivalent to what was there before?

Reply:
Yes, I overlooked the change when working on another solution. This should work fine without the change.

Reply:
On testing, I now recall that the reason for this was to pass a list of selected plugins to the state class. This state class then grabbed each one and singly added them to a list and added that list as an entry to the activeSelectablePlugins list.
Regarding the double list structure, and your comment in the original issue post; Would you like this refactored, or the implementation changed to match the design decision of including multiple plugins in a single list to suit the question and tabbed design?

Reply:
Yes I think that would be a better design.